invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/menu.overrides

```diff
@@ -220,75 +224,62 @@
 }
 
 @media all and (min-width: @computerBreakpoint) {
-  #invenio-nav {
-    #rdm-burger-toggle {
-      display: none;
-    }
+  #invenio-nav #rdm-burger-toggle,
+  #invenio-menu #rdm-close-burger-menu-icon {
+    display: none;
   }
 }
 
 /* menu icon (Hamburger icon) */
-.rdm-menu-icon {
-  cursor: pointer;
-  display: inline-block;
-  float: right;
-  padding: 2rem 1.5rem;
-  position: relative;
-  user-select: none;
-}
 
-.rdm-menu-icon .navicon {
-  background: #333;
-  display: block;
-  height: 2px;
-  position: relative;
-  transition: background 0.2s ease-out;
-  width: 18px;
+#rdm-burger-menu-icon,
+#rdm-close-burger-menu-icon {
+  .navicon {
+    background: #333;
+    display: block;
+    height: 2px;
+    position: relative;
+    transition: background 0.2s ease-out;
+    width: 18px;
+
+    &:before,
+    &:after {
+      display: block;
+      background: #333;
+      content: "";
+      height: 100%;
+      position: absolute;
+      transition: all 0.2s ease-out;
+      width: 100%;
+    }
+  }
 }
 
-.rdm-menu-icon .navicon:before,
-.rdm-menu-icon .navicon:after {
-  background: #333;
-  content: "";
-  display: block;
-  height: 100%;
+#rdm-burger-toggle {
+  font-size: 20px;
   position: absolute;
-  transition: all 0.2s ease-out;
-  width: 100%;
-}
-
-.rdm-menu-icon .navicon:before {
-  top: 5px;
-}
-
-.rdm-menu-icon .navicon:after {
-  top: -5px;
-}
-
-/* menu btn */
-
-.rdm-menu-btn {
-  display: none;
-}
-
-.rdm-menu-btn:checked ~ .menu {
-  max-height: 240px;
-}
-
-.rdm-menu-btn:checked ~ .rdm-menu-icon .navicon {
-  background: transparent;
-}
-
-.rdm-menu-btn:checked ~ .rdm-menu-icon .navicon:before {
-  transform: rotate(-45deg);
-}
-
-.rdm-menu-btn:checked ~ .rdm-menu-icon .navicon:after {
-  transform: rotate(45deg);
-}
+  right: -1rem;
```

Comment:
👀 I guess the negative values were not avoidable ?

Reply:
It's needed to remain the clickable area of the button and aligning with the content below. In fact this value was off, it should be `-1.5rem`.

<img width="148" alt="Screenshot 2022-04-29 at 11 40 52" src="https://user-images.githubusercontent.com/21052053/165921981-c95d417b-306c-474d-9d6e-3b658309c875.png">
<img width="269" alt="Screenshot 2022-04-29 at 11 45 56" src="https://user-images.githubusercontent.com/21052053/165922021-9436879f-ec3a-44ef-83a4-314a00407673.png">